citi-onboarding/virtu-onepage
# Feature/contact section

### What I did
- Made the Contact Section work responsively.
- Made the input component to make the code more organized

### How to test
1. Enter client folder with `cd client`
2. Execute the command `yarn` to download the project modules
3. Execute the command `yarn start` to start the server
3. Open your browser on `http://localhost:3000/`
4. See if the contact section made is responsive and equivalent to the mockup

Resolves #42
